README.md

```diff
@@ -53,6 +58,25 @@ source 'https://rubygems.org'
 gem 'meilisearch'
 ```
 
+`meilisearch` uses some method of Hash supported by Ruby version >= 2.4.0 ([`compact` method](https://github.com/meilisearch/meilisearch-ruby/blob/master/lib/meilisearch/http_request.rb#L17)). So you can use `meilisearch` with Ruby 2.4.0, it works well. But Ruby 2.4.0 is not supported anymore, Ruby 2.5.0 is nearing its end of life. We don't want to support them now. So you can't install `meilisearch` from [https://rubygems.org](https://rubygems.org) if you use Ruby version <= 2.5.x. I prefer you should upgrade your ruby version.
+
+If you still want to use Ruby version <= 2.5.x, you can fork and try to build local gem with this command:
+
+```bash
+> # First, remove this line `s.required_ruby_version = '>= 2.6.0'`
+> gem build meilisearch.gemspec
+  Successfully built RubyGem
+  Name: meilisearch
+  Version: 0.15.0
+  File: meilisearch-0.15.0.gem
+
+> gem install meilisearch-0.15.0.gem
+  Successfully installed meilisearch-0.15.0
+  Parsing documentation for meilisearch-0.15.0
+  Done installing documentation for meilisearch after 0 seconds
+  1 gem installed
+```
+
```

Comment:
I'm not sure this is useful in the README, this makes it too complex, and the README needs to stay simple 🙂